### PR TITLE
Gemini update fine tuning

### PR DESCRIPTION
**Pull request by BenoitDherin**

### CODE REVIEW

File: notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb [hunk 1]
```diff
@@ -0,0 +1,1424 @@
+{
```
Comment (sanjanalreddy) [inline notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb:1]: <p>This line should be in markdown, right now it's rending as code so throws as error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (sanjanalreddy) [inline notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb:1]: <p>Add a line here mention that the tuning takes time so folks are not surprised if it takes multiple hours. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (sanjanalreddy) [reply notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb:1]: You can mention that for epochs = 1 and adapter size = 1 it should take 1 hour to finish fine tuning.

File: notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb [hunk 2]
```diff
@@ -0,0 +1,1440 @@
...
```
Comment (sanjanalreddy) [inline notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb:1]: <p>Add a line here saying this step takes ~5 mins. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

**Review by sanjanalreddy — APPROVED** [review]
LGTM

File: notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb (hunk 1 above)
Comment (BenoitDherin) [reply notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb:1]: <p>Good catch! Done!</p>

Comment (BenoitDherin) [reply notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb:1]: <p>Great! Thanks! Done!</p>

File: notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb (hunk 2 above)
Comment (BenoitDherin) [reply notebooks/vertex_genai/solutions/vertex_llm_tuning_with_gemini.ipynb:1]: <p>Done!</p>

**Pull request merged**